Feat : add useIsTextTruncated hook

DGouron opened the pull request:
Add a hook who return true if the text is truncated.

DGouron committed:
Feat : add useIsTextTruncated hook

maximilienGilet commented:
Hey thanks for the contribution !

The type definition in the JSDoc is missing for the `ref`

Also, could you provide an example please ?

You simply have to create a `UseIsTextTruncatedExample.tsx` file in the `src/examples` directory.

Then you can check the result running `npm run dev`.

I'm currently writing the contributing guide.

Thanks again

DGouron committed:
Fix : ref type in JSDoc and add exemple for the hook

DGouron commented:
@maximilienGilet Thanks for your comment. I've updated the JSDoc and added an example of how to use the hook. I didn't know Astro but it's pretty cool to use. :)
